.github copy/list.rb

```diff
@@ -0,0 +1,17 @@
+require  'enumerable'
+
+class List
+    include  enumerable
+
+    def initialize(*list)
+        @list = list
+    end
```

Comment:
- [ ] Optional suggestions

- `#initialize` method accepts multiple arguments. so I would recommend doing this

````
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```
`